docs/framework/tutorial/shard.md

```diff
@@ -349,8 +349,10 @@ Then create a directory to hold the output, and run `basin-hatch` on the hatch d
 
 ```bash
 mkdir -p content
-basin-hatch hatch_cc-blog_20180226_082238 content
+basin-hatch --no-create-sets hatch_cc-blog_20180226_082238 content
 ```
+The `--no-create-sets` option is because we want to categorize our content ourselves, instead of letting `basin-hatch` do it based on the Wordpress tags from the ingester.
```

Comment:
Just a reminder that the PR for this feature in basin, still needs to be updated.